pkg/flag/db_flags.go

```diff
@@ -64,7 +65,7 @@ var (
 		Name:       "light",
 		ConfigName: "db.light",
 		Usage:      "deprecated",
-		Deprecated: true,
+		Deprecated: `This flag is ignored.`,
```

Comment:
Perhaps we can mark this flag as `Removed`?

Reply:
It will be a breaking change as a command with `--light` will abort. I want to remove it separately. We probably need another announcement for the breaking change.

Reply:
Got it!